warehouse/integrations/deltalake-native/deltalake.go

```diff
@@ -191,6 +197,7 @@ func (d *Deltalake) connect() (*sqlmiddleware.DB, error) {
 			warehouseutils.GetConfigValue(catalog, d.Warehouse),
 			"",
 		),
+		dbsql.WithRetries(d.Retries, d.RetryMaxWait, d.RetryMinWait),
```

Comment:
These seem to be mixed up as well, the 2nd parameter should be min wait and the 3rd max wait.

```go
func WithRetries(retryMax int, retryWaitMin time.Duration, retryWaitMax time.Duration) connOption {
	return func(c *config.Config) {
		c.RetryWaitMax = retryWaitMax
		c.RetryWaitMin = retryWaitMin
		c.RetryMax = retryMax
	}
}
```

Reply:
```suggestion
		dbsql.WithRetries(d.Retries, d.RetryMinWait, d.RetryMaxWait),
```